docs/par2cron_bundle.md

```
@@ -0,0 +1,48 @@
+without user interaction upon detecting corrupted metadata.
+
+Normally bundle files are packed during "create" operations
+where --bundle was given or set as a marker directive. Some
+situations may require to pack existing PAR2 sets or unpack
+existing par2cron bundles, which these commands offer to do.
+
+Full documentation at: https://github.com/desertwitch/par2cron
+
+### Options
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)